.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+### Fixes #ISSUE_NUMBER
+
+### Description
+
+Include a summary of the change and relevant motivation/context. Do mention the dependencies that have been added (if any).
+
+**Type of Change:** (Delete irrelevant options)
+- Code
+- Testing
+- User Interface
+- Documentation
+
+**Code/Quality Assurance Only** (Delete irrelevant options)
+
+- Bug fix (non-breaking change which fixes an issue)
+- This change requires a documentation update (software upgrade on readme file)
+- New feature (non-breaking change which adds functionality pre-approved by mentors)
+
+
+### How Has This Been Tested?
+
+Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.
+
+
+### Checklist:
+
+- [ ] My PR follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code or materials
+- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] Any dependent changes have been merged
+- [ ] I have added one/more required dependencies
+
+
+**Code/Quality Assurance**
+- [ ] My code follows TDD clean architecture
+- [ ] My changes generate no new warnings 
+- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
```

Comment:
Add a section that asks for a list of packages added to the project.
"List of packages / plugins added to the project (If any)"